lib/resources.yml

```diff
@@ -63,11 +63,13 @@ js:
       - angular2: '*'
     React (latest):
       - react: '*'
+      - react-dom: '*'
 
   sourceFormat:
     options:
       - ES5
       - ES6
+      - JSX+ES6
```

Comment:
What if we add a question, "Do you wish to write HTML code inside your JS components? (JSX/TSX support)". This sounds like something I don't really want to do, but some people do. Then with that answer (let's call it `buildJS.supportJSX`), you can make the appropriate modifications.

On a side note, I'm finding a need for a 'jsExtensions' property, when trying to support *.js then *.ts source code. jsx is an extension, but it also affects compiler options and loaders.